ansible/roles/jenkins-worker/tasks/main.yml

```diff
@@ -289,7 +328,7 @@
     name: java-base
 
 - name: enable jenkins at startup - general
-  when: not os|startswith("zos") and not os|startswith("macos") and not os|startswith("aix")
+  when: not os|startswith("zos") and not os|startswith("macos") and not os|startswith("aix") and not os|startswith("ibmi")
```

Comment:
I believe ansible also lets you write this like this:
```yaml
when:
  - not os|startswith("aix")
  - not os|startswith("ibmi")
  - not os|startswith("macos")
  - not os|startswith("zos")
```
Which is arguably more readable.

Reply:
Change made. We'll flush out when we go to deploy if there are any issues. Thanks!